wire-ios-data-model/Source/MLS/CoreCryptoProvider.swift

```diff
@@ -233,7 +234,7 @@ public actor CoreCryptoProvider: CoreCryptoProviderProtocol {
         }
 
         WireLogger.mls.info("generating ed25519 public key")
-        let keyBytes = try await coreCrypto.perform { try $0.clientPublicKey(ciphersuite: defaultCipherSuite.rawValue) }
+        let keyBytes = try await coreCrypto.perform { try await $0.clientPublicKey(ciphersuite: CiphersuiteName.default.rawValue) }
```

Comment:
Would making the `CiphersuiteName` be part of the initialiser to make it configurable?

Reply:
Yes, that would probably make sense when we need that in the future, maybe for e2ei?

Reply:
Let's deal with this when we actually need configurable ciphersuites, I don't want to guess on what API we will need in the future.